### PR TITLE
store the Revision we get from the server in the locally store manifest

### PR DESCRIPTION
**Pull request by mvo5**
Trivial branch that just stores the revision in our local store manifest.

### CODE REVIEW

File: snap/remote/remote.go [hunk 1]
```diff
@@ -39,6 +42,7 @@ type Snap struct {
 	Prices          map[string]float64 `json:"prices,omitempty"`
 	Publisher       string             `json:"publisher,omitempty"`
 	RatingsAverage  float64            `json:"ratings_average,omitempty"`
+	Revision        string             `json:"revision"`
```
Comment (niemeyer) [inline snap/remote/remote.go]: Is it a string!?

Comment (pedronis) [inline snap/remote/remote.go]: no, it's actually an integer:
curl https://search.apps.ubuntu.com/api/v1/package/hello-world.mvo

... "revision": 1, ...

Comment (mvo5) [inline snap/remote/remote.go]: Silly me, thanks for catching this!

**Comment by chipaca** [comment]
Wy u no tests?
(hint: `grep -r support_url`)

File: snap/remote/remote.go [hunk 2]
```diff
@@ -24,6 +24,9 @@ import (
 )
 
 // A Snap encapsulates the data sent to us from the store.
+//
+// Full json available via:
+// curl -s -H "accept: application/hal+json" -H "X-Ubuntu-Release: 15.04-core" https://search.apps.ubuntu.com/api/v1/package/8nzc1x4iim2xj1g2ul64.chipaca | python -m json.tool
```
Comment (niemeyer) [inline snap/remote/remote.go]: I get an error with this command.

Comment (mvo5) [inline snap/remote/remote.go]: Thanks! I updated the command so that it works again.

**Comment by mvo5** [comment]
@chipaca I'm not sure I understand the sentence. I think you mean I should update the example and server mock replies as well (?). If that is the case I did so now. It might mean that there is no test for this? If I failed at the "guess the meaning" game, please let me know and I will play another round ;)

**Comment by niemeyer** [comment]
LGTM, pending on @chipaca's point.

**Comment by chipaca** [comment]
I meant, there were absolutely no tests for this. I wanted some kind of testering. Now I have some, so I'm happy.

**Pull request merged**